docs/resources/integration_gcp_sts_account.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Out of curiosity, is this the documentation that gets displayed on the Terraform website? If not, what controls that? 

Just wondering because I think it could be helpful to include a few examples of setting up this integration (which will also involve setting up some GCP service accounts and their config with Terraform)